### PR TITLE
feat: Support PortablePDB embedded sources

### PR DESCRIPTION
**Pull request by vaind**
Followup from https://github.com/getsentry/symbolic/pull/734 now that symbolic got released,
this enables source context support for users uploading Portable PDBs built with `<EmbedAllSources>true</EmbedAllSources>` project property.

- chore: update symbolic to v11.0.0
- test: portable PDB with embedded sources
- feat: support portable PDB embedded sources

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2892,7 +2892,7 @@ dependencies = [
  "serde",
  "serde_json",
  "structopt",
- "symbolic-common",
+ "symbolic-common 10.2.1",
```
Comment (Swatinem) [inline Cargo.lock:2895]: pls also bump the version in the `process-event` utility :-)

File: crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap [hunk 2]
```diff
@@ -49,4 +49,8 @@ modules:
         location: "http://localhost:<port>/symbols/bd/a18fd85d4a4eb893022d6bfad846b1.src.zip"
         download:
           status: notfound
+      - source: local
+        location: "http://localhost:<port>/symbols/demo.wasm/BDA18FD85D4A4EB893022D6BFAD846B10/demo.wasm"
```
Comment (Swatinem) [inline crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap:53]: I’m wondering why a completely unrelated filetype now has more candidates? 🤔

Comment (vaind) [reply crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap:53]: yeah that was weird but my assumption is that it's related to symbolic update

File: Cargo.lock (hunk 1 above)
Comment (vaind) [reply Cargo.lock:2895]: done

File: crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap (hunk 2 above)
Comment (Swatinem) [reply crates/symbolicator-service/tests/integration/snapshots/integration__symbolication__wasm_payload.snap:53]: possible. trying yet another file name pattern does definitely not hurt, though I would still love to find out why this suddenly happens.

**Review by Swatinem — APPROVED** [review]

**Pull request merged**